### PR TITLE
{#49} add one InlineKeyboard know-don't know

### PR DESCRIPTION
**Pull request by UlkaBo**
add one InlineKeyboard know-don't know
add two function 'my variant' 'nothing to do'

Signed-off-by: UlkaBo <ul.bozhko@gmail.com>

### CODE REVIEW

File: app/handlers/personal/start.py [hunk 1]
```diff
@@ -159,6 +159,14 @@ async def add_words_to_study(callback_query: types.CallbackQuery, callback_data:
         await callback_query.answer('Added to study.')
 
 
+async def my_variant(callback_query: types.CallbackQuery):
```
Comment (KonstZiv) [inline app/handlers/personal/start.py:162]: Function is missing a return type annotation  [no-untyped-def] mypy(error)

File: app/handlers/personal/start.py [hunk 2]
```diff
@@ -159,6 +159,14 @@ async def add_words_to_study(callback_query: types.CallbackQuery, callback_data:
...
+    await callback_query.answer('it is example')
+
+
+async def nothing_to_do(callback_query: types.CallbackQuery):
```
Comment (KonstZiv) [inline app/handlers/personal/start.py:166]: Function is missing a return type annotation  [no-untyped-def] mypy(error)

File: app/handlers/personal/start.py [hunk 3]
```diff
@@ -169,4 +177,6 @@ def register_handler_start(dp: Dispatcher):
         select_target_language, FSMChooseLanguage.target_language
     )
     dp.callback_query.register(add_words_to_study, ToStudyCallbackData.filter())
+    dp.callback_query.register(my_variant, F.data == 'my_variant')
+    dp.callback_query.register(nothing_to_do, F.data == 'nothing_to_do')
     dp.message.register(translate_text)  # F.test.regexp("[a-zA-Z ]"))
```
Comment (KonstZiv) [inline app/handlers/personal/start.py:182]: In fact, I have no comments on the essence of the code, but there are comments on the form. The code does not pass type checks (I indicated in two places, but in total, in one form or another, the linter showed me 35 errors in start.py). Basically, these are simple errors that are easy to fix.
I suggest making this a rule.
I think that it is necessary to include automatic mypy check in the CI process, but after correcting all the errors in the code. Just a separate task for the next sprint.

**Review by KonstZiv — CHANGES_REQUESTED** [review]

File: app/handlers/personal/callback_data_states.py [hunk 4]
```diff
@@ -11,3 +11,11 @@ class ToStudyCallbackData(CallbackData, prefix='to_study'):
...
     '''
     item_relation_id: UUID
+
+
+class StudyCardCallbackData(CallbackData, prefix='study_card'):
+    '''
+    CallbackQuerry  - "know", "don't know"
+    data - 'card_id'
+    '''
+    card_id: UUID
```
[the remark this answers is not in the document]
Comment (UlkaBo) [reply app/handlers/personal/callback_data_states.py:21]: fixed

File: app/handlers/personal/start.py (hunk 1 above)
Comment (UlkaBo) [reply app/handlers/personal/start.py:162]: add -> None

File: app/handlers/personal/start.py (hunk 2 above)
Comment (UlkaBo) [reply app/handlers/personal/start.py:166]: add -> None

**Review by KonstZiv — APPROVED** [review]

**Pull request merged**